cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/action/Action.java

```diff
@@ -30,4 +31,14 @@
    * @throws Exception when there is failure in method execution
    */
   public abstract void run(ActionContext context) throws Exception;
+
+  /**
+   * Implement this method to validate configurations
```

Comment:
Do we need a javadoc for this? The method defined in the interface already has a javadoc, right